### PR TITLE
Add instructions in README

### PR DESCRIPTION
**Pull request by lucie-docs**
Hi @hectorsector,

Added the instructions for this exercise.

A couple of things:

- I have made instruction 2 more precise ("Create the necessary file") than what Matt had originally written ("Configure secret scanning") to avoid having the learner trying to enable secret scanning on the repository (or see if it is enabled) which would normally be the first step to take before doing anything. I thought it's either giving away that the learner has to create a file or mentioning somewhere that the learner does not need to enable secret scanning for this exercise, thoughts on this?
- I wanted to take a screenshot of the troubleshooting workflow message but it seems it's not returning anything, is that normal?
- I got this email as soon as I made a copy of the exercise and after I finished it:
Screenshot 2021-11-16 at 14 50 25
Where is this coming from? Should the learner be receiving these?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,29 +37,32 @@ Please complete the instructions below:
 
 1. Create your own copy of this repository by using the [Use this template](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template#creating-a-repository-from-a-template) button.
 
-<!-- Add your steps below starting with step 2 -->
+2. Create the necessary file in the repository to exclude the following files from being scanned:
+  - Any file in a `docs` directory anywhere in the repository.
+  - All `js` files in a `dist` folder at the root of the repository.
```
Comment (hectorsector) [inline README.md]: This indent is showing up funky

**Review by hectorsector — COMMENTED** [review]
Thanks, @lucie-docs! 

> I have made instruction 2 more precise ("Create the necessary file") than what Matt had originally written ("Configure secret scanning") to avoid having the learner trying to enable secret scanning on the repository (or see if it is enabled) which would normally be the first step to take before doing anything. I thought it's either giving away that the learner has to create a file or mentioning somewhere that the learner does not need to enable secret scanning for this exercise, thoughts on this?

It looks great to me the way you wrote it!

> I wanted to take a screenshot of the troubleshooting workflow message but it seems it's not returning anything, is that normal?

That's probably because there's [no troubleshooting message](https://github.com/githubtraining/exercise-exclude-files-from-secret-scanning/blob/main/.github/workflows/grading.yml#L27-L29). I can add one to this PR if you don't mind me committing to your branch. 

I also stopped putting screenshots into READMEs so that there's one less thing we have to update. We'll still have to fix the issue with the troubleshooting message, though. 

> I got this email as soon as I made a copy of the exercise and after I finished it:

I believe that is unit tests for the action itself, I'm not sure why it's failing but it's unrelated to the instructions, I think.

**Comment by lucie-docs** [comment]
@hectorsector I fixed the indent and deleted the screenshot, feel free to add the troubleshooting message to this PR!

**Review by hectorsector — APPROVED** [review]
Thanks again for the help, @lucie-docs! This is ready to 🚢

**Pull request merged**